agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -616,11 +618,35 @@ async def create_deployment(
             raise Exception(f"Error while creating deployment: {e}")
 
 
+async def get_app_type_from_template(template_id: Optional[str]) -> Enum:
```

Comment:
`get_app_type_from_template` -> `get_app_type_from_template_id` because for a non-template app, the `template_id` cam None but there can be no `template`.

Reply:
Thank you.